recipes/py-libnuma/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+context:
+  version: '1.2'
+
+package:
+  name: py-libnuma
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/p/py-libnuma/py-libnuma-${{ version }}.tar.gz
+    sha256: e9d67ad04f274015e9acac497d642ce92f8a96d1d4e7be24e39124b24320b95a
+
+build:
+  number: 0
+  script: ${{ PYTHON }} -m pip install .
+  skip:
+    - win
+    - osx
```

Comment:
This one should probably be OS-noarch. No idea on how to do that in the v1 recipe format though.

Reply:
Please see https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies